go.mod

```diff
@@ -233,7 +235,7 @@ require (
 	go.uber.org/atomic v1.11.0
 	go.uber.org/automaxprocs v1.5.3
 	go.uber.org/dig v1.17.0
-	go.uber.org/fx v1.18.2
+	go.uber.org/fx v1.20.1
```

Comment:
In the past we had issue with 1.20.1 breaking some dogstatsd behavrior. You might want to sync with the agent-platform team.

Reply:
It is not my intent to update this. It was miss. Reverted this.